src/GA_script.ts

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
I think it would be better to just paste the GA setup into App.svelte because it would be easier to update if the GA API changes.
If you want to keep this file, it needs a license block at the top.

Reply:
pasted it directly into index.html in the head tag.